src/components/team/DomainDesktop.tsx

```diff
@@ -13,7 +13,7 @@ const DomainDesktop = ({
   members,
 }: {
   domainName: string
-  members: any[]
+  members: Members[]
```

Comment:
This looks good! I think however that Members should be singular, not plural.

This naming would of course conflict with the Member component. Maybe we should rename that component to MemberCard?

What do you think?

Reply:
Got it! It has been updated!